lib/core/src/lib/datatable/components/datatable/datatable.component.ts

```diff
@@ -699,6 +699,13 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
         }
     }
 
+    onCheckboxLabelClick(row: DataRow, event: MouseEvent) {
+        const target = event.target as HTMLElement;
+        if (!(target.tagName === 'MAT-CHECKBOX' || target.closest('mat-checkbox'))) {
```

Comment:
can we have some data-attribute instead? otherwise we hardcode the material checkbox there, and things might break later if we change the component

Reply:
good point, replaced with attribute, thanks